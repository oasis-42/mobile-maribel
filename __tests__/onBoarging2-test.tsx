import { render, screen } from "expo-router/testing-library";
import OnBoarding2 from "../src/app/screens/onboardings/onBoarding2";

test("should render correctly", () => {
  render(<OnBoarding2 />);
  expect(screen.toJSON()).toMatchSnapshot();
});
