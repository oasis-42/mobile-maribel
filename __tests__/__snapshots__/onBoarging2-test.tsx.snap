// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    collapsable={false}
    pointerEvents="box-none"
    style={
      {
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      style={
        {
          "flexDirection": "column",
        }
      }
    >
      <View>
        <View
          style={
            {
              "gap": 14,
              "marginTop": 12,
              "padding": 10,
            }
          }
        >
          <Text
            style={
              {
                "fontSize": 25,
                "fontWeight": "700",
              }
            }
          >
            Personalize sua evolução
          </Text>
          <Text
            style={
              {
                "fontWeight": "600",
              }
            }
          >
            Entenda quais formas de avaliação melhor se encaixam em seus estudos
          </Text>
          <View
            collapsable={false}
            style={
              {
                "backgroundColor": "rgba(247, 243, 249, 1)",
                "borderRadius": 12,
                "shadowColor": "#000",
                "shadowOffset": {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.15,
                "shadowRadius": 3,
              }
            }
            testID="card-container-outer-layer"
          >
            <View
              collapsable={false}
              style={
                {
                  "backgroundColor": "rgba(247, 243, 249, 1)",
                  "borderColor": "#D7D7D7",
                  "borderRadius": 12,
                  "elevation": 1,
                  "flex": undefined,
                  "shadowColor": "#000",
                  "shadowOffset": {
                    "height": 1,
                    "width": 0,
                  },
                  "shadowOpacity": 0.3,
                  "shadowRadius": 1,
                }
              }
              testID="card-container"
            >
              <View
                style={
                  [
                    {
                      "flexShrink": 1,
                    },
                    undefined,
                  ]
                }
                testID="card"
              >
                <View
                  style={
                    [
                      {
                        "alignItems": "center",
                        "flexDirection": "row",
                        "justifyContent": "space-between",
                        "paddingLeft": 16,
                      },
                      {
                        "minHeight": 50,
                      },
                      undefined,
                    ]
                  }
                >
                  <View
                    style={
                      [
                        {
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        },
                      ]
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        [
                          {
                            "textAlign": "left",
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                            "writingDirection": "ltr",
                          },
                          [
                            {
                              "fontFamily": "System",
                              "fontSize": 16,
                              "fontWeight": "400",
                              "letterSpacing": 0.15,
                              "lineHeight": 24,
                            },
                            [
                              {
                                "minHeight": 30,
                                "paddingRight": 16,
                              },
                              {
                                "marginBottom": 2,
                              },
                              {
                                "fontSize": 20,
                                "fontWeight": "700",
                              },
                            ],
                          ],
                        ]
                      }
                    >
                      Avaliação guiada
                    </Text>
                  </View>
                  <View />
                </View>
                <View
                  borderRadiusStyles={{}}
                  style={
                    [
                      {
                        "paddingHorizontal": 16,
                      },
                      {
                        "paddingBottom": 16,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text
                    style={
                      {
                        "fontSize": 12,
                        "fontWeight": "400",
                      }
                    }
                  >
                    Faça a sua avaliação com maior assertividade ao seguir nossos temas e instruções predefinidos.
                  </Text>
                </View>
              </View>
            </View>
          </View>
          <View
            collapsable={false}
            style={
              {
                "backgroundColor": "rgba(247, 243, 249, 1)",
                "borderRadius": 12,
                "shadowColor": "#000",
                "shadowOffset": {
                  "height": 1,
                  "width": 0,
                },
                "shadowOpacity": 0.15,
                "shadowRadius": 3,
              }
            }
            testID="card-container-outer-layer"
          >
            <View
              collapsable={false}
              style={
                {
                  "backgroundColor": "rgba(247, 243, 249, 1)",
                  "borderColor": "#D7D7D7",
                  "borderRadius": 12,
                  "elevation": 1,
                  "flex": undefined,
                  "shadowColor": "#000",
                  "shadowOffset": {
                    "height": 1,
                    "width": 0,
                  },
                  "shadowOpacity": 0.3,
                  "shadowRadius": 1,
                }
              }
              testID="card-container"
            >
              <View
                style={
                  [
                    {
                      "flexShrink": 1,
                    },
                    undefined,
                  ]
                }
                testID="card"
              >
                <View
                  style={
                    [
                      {
                        "alignItems": "center",
                        "flexDirection": "row",
                        "justifyContent": "space-between",
                        "paddingLeft": 16,
                      },
                      {
                        "minHeight": 50,
                      },
                      undefined,
                    ]
                  }
                >
                  <View
                    style={
                      [
                        {
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        },
                      ]
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        [
                          {
                            "textAlign": "left",
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                            "writingDirection": "ltr",
                          },
                          [
                            {
                              "fontFamily": "System",
                              "fontSize": 16,
                              "fontWeight": "400",
                              "letterSpacing": 0.15,
                              "lineHeight": 24,
                            },
                            [
                              {
                                "minHeight": 30,
                                "paddingRight": 16,
                              },
                              {
                                "marginBottom": 2,
                              },
                              {
                                "fontSize": 20,
                                "fontWeight": "700",
                              },
                            ],
                          ],
                        ]
                      }
                    >
                      Avaliação adaptativa
                    </Text>
                  </View>
                  <View />
                </View>
                <View
                  borderRadiusStyles={{}}
                  style={
                    [
                      {
                        "paddingHorizontal": 16,
                      },
                      {
                        "paddingBottom": 16,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text
                    style={
                      {
                        "fontSize": 12,
                        "fontWeight": "400",
                      }
                    }
                  >
                    Faça a sua avaliação com novas percepções ao deixar a inteligência artificial sem instruções.
                  </Text>
                </View>
              </View>
            </View>
          </View>
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            href="./onBoarding3"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            role="link"
            style={
              {
                "backgroundColor": "#044884",
                "flex": 1,
                "height": 56,
                "justifyContent": "center",
                "padding": 8,
                "width": "auto",
              }
            }
          >
            <Text
              style={
                {
                  "color": "#FFFFFF",
                  "fontWeight": "600",
                  "textAlign": "center",
                }
              }
            >
              Continuar
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RNCSafeAreaProvider>
`;
